src/host/ut_host/VtIoTests.cpp

```diff
@@ -360,7 +362,7 @@ void VtIoTests::RendererDtorAndThread()
         // EnablePainting gets called, and if that happens, then the thread will
         // never get destructed. This will only ever happen in the vstest test runner,
         // which is what CI uses.
-        Sleep(500);
+        /*Sleep(500);*/

```

Comment:
stray comment. Delete?
(and below)

Reply:
Gonna leave it for now because these tests are notorious.